docs/topics/modules/creating_new_project.adoc

```diff
@@ -36,20 +37,25 @@ image::project_springboot.png[Spring Boot HTTP Project]
 .. In the `OpenShift Project Name` field, add *myspringboot* as the name for your project.
 endif::[]
 // conditional ends here
-.. Leave the default value for the `Organisation`,`Group Id`, and `Version` fields and click btn:[Next].
-.. Select a build pipeline strategy. For this example, use the default value (`Release, Stage, Approve and Promote`) and click btn:[Next].
+.. Keep the default value for the `Organisation`, `Group Id`, and `Version` fields and click btn:[Next].
+.. Select a build pipeline strategy. For this example, use the default value `Release, Stage, Approve and Promote` and click btn:[Next].
 +
 NOTE: Pipelines allow developers to create a repeatable, reliable, and incrementally improving process to move the software from code to the execution environment.
 +
-.. The `Build Config` screen displays the details for your `Jenkins Space` and options for web hooks and triggering builds. Use the default options for these fields and click btn:[Finish].
+
+.. The `Build Config` screen displays the details for your `Jenkins Space` and options for adding continuous integration hooks and triggering a build when your application is deployed. Use the default options for these fields and click btn:[Finish].
 .. The `Summary` screen displays your selected options. Click btn:[Ok] to continue.
 
 // for hello-world
 ifeval::["{context}" == "hello-world"]
-Your new Vert.X project is now created in your Space. {osio} now adds a pipeline into your OpenShift Online account and creates a new GitHub repository for your project.
+Your new Vert.X project is now created in your Space.
```

Comment:
Is there a reason this was removed? Creating a new project triggers a pipeline build and creates the codebase in GitHub, which is probably useful to know.

Reply:
See, the next paragraph, I have added that info and a bit more in the section which is relevant for both vertx and springboot. I have just removed it from the conditional for springboot.

Reply:
Ok, thanks. I can't see where it's moved to in this commit so assumed it was removed.

Reply:
Last line: {osio} has now hosted the project source code in the linked account and specified organization on GitHub. It  has also hosted the pipeline for the project in OpenShift Online.